foo.js

```diff
@@ -94,6 +79,15 @@ class Stargazers extends React.Component {
   }
 }
 
+Stargazers = createContainer(Stargazers, {
+  listenTo: UserStore,
+  props: {
+    users() {
```

Comment:
This'd feel cleaner to me if users() was passed props as a parameter instead of using `this.props`